packages/token-dictionary/__tests__/spacing.test.ts

```diff
@@ -144,6 +149,7 @@ test('with semantic spacing', () => {
             "sm",
           ],
           "prop": "-sm",
+          "rawValue": "calc(var(--spacing-sm) * -1)",
```

Comment:
no strong opinion here but I this almost feels like there's a need for another property `resolvedValue: "calc(var(--spacing-sm) * -1)",` with `rawValue: "-40px"`.. but this might be weird since `rawValue` would never actually be used (internally, at least)

Reply:
I tried to quickly implement this but it's a bit complex, maybe I should tackle this in the separate PR.